package.json

```diff
@@ -38,45 +41,50 @@
     "vue": ">=2.7.14"
   },
   "dependencies": {
-    "@vueuse/core": "9.13.0",
-    "@vueuse/shared": "9.13.0",
-    "vue-demi": "0.13.11"
-  },
-  "devDependencies": {
-    "@babel/core": "7.21.3",
-    "@babel/types": "7.21.3",
-    "@rotki/eslint-config": "1.0.4",
-    "@storybook/addon-essentials": "7.0.0-rc.8",
-    "@storybook/addon-interactions": "7.0.0-rc.8",
-    "@storybook/addon-links": "7.0.0-rc.8",
-    "@storybook/addon-styling": "next",
-    "@storybook/blocks": "7.0.0-rc.8",
-    "@storybook/testing-library": "0.0.14-next.1",
-    "@storybook/vue": "7.0.0-rc.8",
-    "@storybook/vue-vite": "7.0.0-rc.8",
-    "@types/node": "16.18.12",
-    "@vitejs/plugin-vue": "4.1.0",
+    "@babel/core": "7.21.4",
+    "@babel/types": "7.21.4",
+    "@rotki/eslint-config": "1.0.6",
+    "@storybook/addon-essentials": "7.0.7",
+    "@storybook/addon-interactions": "7.0.7",
+    "@storybook/addon-links": "7.0.7",
+    "@storybook/addon-styling": "1.0.5",
+    "@storybook/blocks": "7.0.7",
+    "@storybook/testing-library": "0.1.0",
+    "@storybook/vue": "7.0.7",
+    "@storybook/vue-vite": "7.0.7",
+    "@types/node": "18.16.1",
+    "@vitejs/plugin-vue": "4.2.1",
```

Comment:
Is there any specific reason why we would need to move the devDepedencies to dependencies? 

The library should not depend directly on storybook we only use it for tooling so I would assume that the proper place would be devDependencies.

Reply:
I'll move them to dev, was having lots of challenges when building, so I moved them temporarily, will return the indirect dependencies to dev